kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -23,6 +23,14 @@ def __init__(self, J, shape, Q=1, T=None, max_order=2, average=True,
         self.out_type = out_type
         self.backend = backend
 
+        if not average:
+            warn("The average option is deprecated and will be "
+                 "removed in v0.4."
+                 "For average=True, set T=None for default averaging,"
+                 "or T>1 (positive integer) for custom averaging."
+                 "T=0 with average=False will not average.",
+                 DeprecationWarning)
```

Comment:
`if average is not None`

Reply:
if the user ignores specifying an average that means they've switched over. If `average` is False, the user specified this then.